README.md

```diff
@@ -21,7 +21,7 @@ standard library, and documentation.
 
 ## Why Rust?
 
-- **Performance:** Fast and memory-efficient, suitable for critical services, embedded devices, and easily integrate with other languages.
+- **Performance:** Fast and memory-efficient, suitable for critical services and embedded devices, and easily integrated with other languages.
```

Comment:
It might be worth just copying the [website](https://www.rust-lang.org/) text, the updates here don't make this sound right to me. 

```suggestion
- **Performance:** Rust is blazingly fast and memory-efficient: with no runtime or garbage collector, it can power performance-critical services, run on embedded devices, and easily integrate with other languages. 
```

Reply:
Good idea, thanks :)